packages/config/config/devices/0x0178/th100nx.json

```diff
@@ -0,0 +1,412 @@
+// Nexia Home Intelligence TH100NX
+// Remote Temperature and Humidity Sensor
+{
+	"manufacturer": "Nexia Home Intelligence",
+	"manufacturerId": "0x0178",
+	"label": "TH100NX",
+	"description": "Remote Temperature and Humidity Sensor",
+	"devices": [
+		{
+			"productType": "0x5448",
+			"productId": "0x3130",
+			"zwaveAllianceId": 2223
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"paramInformation": {
+		"1": {
+			"label": "Time Between Battery Reports",
+			"valueSize": 1,
+			"unit": "hours",
+			"minValue": 0,
+			"maxValue": 127,
+			"defaultValue": 0,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true,
+			"options": [
+				{
+					"label": "Disabled",
+					"value": 0
+				}
+			]
+		},
+		"2": {
```

Comment:
Not sure if parameters 2 to 5 should be in `°F`, at least the manual says so.

Reply:
Yeah, I'd left that out because Parameter 7 changes it to celsius.